### DISCUSSION TITLE
Fast Yolov3 CPU Inference Using Onnxruntime

### DISCUSSION THREAD

**Discussion by matankley** [microsoft/onnxruntime#6521] [Show & Tell]
Hi,
I would like to thank you and share with you few insights i observed while testing inference of Yolov3 Object detection model using Onnxruntime on CPU.

Onnxruntime inference is significantly faster than the common opencv-dnn and darknet that are usually used for yolov3 models.

Here is a link to the full article - [(https://medium.com/towards-artificial-intelligence/yolov3-cpu-inference-performance-comparison-onnx-opencv-darknet-6764f2bde33e](url))

Hope this can help someone and show the great value we can earn from onnxruntime.
Thanks !

**Comment by faxu** [comment]
Thank you for sharing! 👍 Was this primarily for research purposes or are you using this solution in some production scenarios?

**Reply by matankley** [reply]
@faxu The primary purpose was to find a solution for production scenarios, so it will be soon introduced in large-scale production.

**Comment by Nuwan1654** [comment]
Is there any comparison for yolov7 ONNX models?
